bazel/cc_wrapper.py

```diff
@@ -6,6 +6,7 @@
 
 envoy_real_cc = {ENVOY_REAL_CC}
 envoy_real_cxx = {ENVOY_REAL_CXX}
+envoy_cxxflags = {ENVOY_CXXFLAGS}
```

Comment:
This will change the content of the compiler wrapper when $CXXFLAGS changes, which will have significant negative impact on build caching.

When the crosstool bootstrap runs this script, does it propagate environment variables? If yes, then you can use `envoy_cxxflags = os.environ.get("CXXFLAGS", "")`

Reply:
Well, that's kind of the point, isn't it? Same is true for changing `$CC` and `$CXX`.

Reply:
Bazel won't cache when flags changed so I don't think it is negative anyway?